badgerdb/badger_graph.go

```diff
@@ -451,6 +474,8 @@ func (bgdb *BadgerGDB) DelVertex(id string) error {
 		}
 		return nil
 	})
+	bgdb.ts.Touch(bgdb.graph)
```

Comment:
If there's an error, the graph didn't change.

Reply:
Offline discussion: not precise, but acceptable behavior